package NYT.NHydra.NProto;

import "yt/core/misc/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TMutationHeader
{
    required string mutation_type = 1;
    optional uint64 timestamp = 2;
    optional uint64 random_seed = 3;
    optional NYT.NProto.TGuid mutation_id = 4;
    required int32 segment_id = 5;
    required int32 record_id = 6;
}

////////////////////////////////////////////////////////////////////////////////

message TPeerInfo
{
    required string address = 1;
    required int32 peer_id = 2;
}

message TCellConfig
{
    required int32 version = 1;
    required int32 size = 2;
    repeated TPeerInfo peers = 3;
}

////////////////////////////////////////////////////////////////////////////////
